docs/modules/general/pages/architecture-overview/shell.adoc

```diff
@@ -1,37 +1,45 @@
 = OneCX Shell
 
+:idprefix:
+:idseparator: -
+
 :angular-integration-interface_url: xref:latest@guides:angular:libraries/angular-integration-interface.adoc
 :integration-interface_url: xref:latest@guides:javascript:libraries/integration-interface.adoc
 :mfe_loading_url: xref:implementation-details/mfe-content-loading/index.adoc
```

Comment:
don't mix separators?

Reply:
Separators here are mixed because `integration-interface` is a whole name of the library.